backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/objcTypes.kt

```diff
@@ -18,7 +19,7 @@ sealed class ObjCType {
             if (attrsAndName.isEmpty()) this else "$this ${attrsAndName.trimStart()}"
 }
 
-class ObjCRawType(
+data class ObjCRawType(
```

Comment:
It was (but no longer is) necessary to ensure the forwardDeclaredClasses set won't contain any duplicates. I left it in here, since I found it surprising that some type classes are already data classes, but others are not. Looked like an oversight to me.
Please advise how you would like to proceed (revert, split off into separate PR, keep).

Reply:
> I left it in here, since I found it surprising that some type classes are already data classes, but others are not.

Ah, I see. This motivation is enough, let's just keep this change as a part of this PR.
I asked mostly to better understand the PR.